optimism.tokenlist.json

```diff
@@ -914,6 +914,28 @@
       "decimals": 18,
       "logoURI": "https://ethereum-optimism.github.io/logos/WETH.png"
     },
+    {
+      "chainId": 1,
+      "address": "0x259ab9b9eab62b0fd98729b97be121073d5b3479",
+      "name": "Erica Social Token",
+      "symbol": "EST",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/EST.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x7b0bcC23851bBF7601efC9E9FE532Bf5284F65d3",
```

Comment:
The minimal proxy contract is not compatible with the standard bridge, unless you've done something in the implementation which is probably here `0xd0ff8ca3a1ce9ff2b5ee71230c9f3759b09cec53` ?

Reply:
I'll give you the Sourcify link for this. [0xd0ff8ca3a1ce9ff2b5ee71230c9f3759b09cec53](https://repo.sourcify.dev/contracts/full_match/10/0xd0ff8ca3A1ce9Ff2B5eE71230c9f3759b09cEC53/)

As you can see, we have stored the l2bridge address!
This is enough... Am I right? 🥶
In the test network, these settings worked well with the standard bridge!

Reply:
If you have a token instance on Kovan, can you add it to the token list as well for testing pls? Also please give my account `0xe3a90926133fef3c58514fA1aAeF75E0998aedD5` some test tokens for me to work with 🙏 
As for the token implementation above, I wonder why you need to use the minimal proxy on L2 at all, while you can just deploy the `L2StandardERC20`?

Reply:
Additionally, I attach the verified contract address!

Optimism kovan L2StandardERC20: 
[0xFF0894145309ba3D8D32c73Fc514CB0e370a0F69](https://repo.sourcify.dev/contracts/full_match/69/0xFF0894145309ba3D8D32c73Fc514CB0e370a0F69/)

Kovan StandardERC20: [0x3121dCF2CB403eaF0575f3D2a819f0902DeCE347](https://repo.sourcify.dev/contracts/full_match/42/0x3121dCF2CB403eaF0575f3D2a819f0902DeCE347/)

Reply:
Is there any reason why the token on Optimism is not just the `L2StandardERC20` instead of the minimal proxy plus implementation?

Reply:
Can you tell me what plus means?
If you're asking why we're using minimal proxies, answer is Optimism is cheap enough, but I think it's good for all of us not to increase the size of the chain in the long run.

I don't know, but is there a problem with the minimal proxy?

Reply:
Right! I agree that readability is important! it is also true that it is difficult to verify the current code to use Optimism Etherscan.

In this case, I needed a permit. simply minimal proxy was an additional factor. Also, I know the details of my token, and I thought it wouldn't be a problem because most of it would be used through the interface.
I hope the test goes well and Thank you for your time for this. 🙏

Reply:
>I needed a permit. simply minimal proxy was an additional factor.

sorry I don't understand this part, can you clarify?

Reply:
Also please verify this in Etherscan so it can be read 0xff0894145309ba3d8d32c73fc514cb0e370a0f69

Reply:
> sorry I don't understand this part, can you clarify?

that's a [eip-2612](https://eips.ethereum.org/EIPS/eip-2612) !

Reply:
> Also please verify this in Etherscan so it can be read 0xff0894145309ba3d8d32c73fc514cb0e370a0f69 as the implementation of the minimal proxy. Thanks

I Verified it! plz check [0xFF0894145309ba3D8D32c73Fc514CB0e370a0F69](https://kovan-optimistic.etherscan.io/address/0xff0894145309ba3d8d32c73fc514cb0e370a0f69)

Reply:
Thanks! Kovan tests okay, if you can verify the mainnet implementation for proxy https://optimistic.etherscan.io/address/0x7b0bcC23851bBF7601efC9E9FE532Bf5284F65d3#code as well please

Reply:
Also note that the EIP-2612 functionality is not included in the `L2StandardERC20` implementation so if you need that it needs to be implemented in addition.

Reply:
> Thanks! Kovan tests okay, if you can verify the mainnet implementation for proxy https://optimistic.etherscan.io/address/0x7b0bcC23851bBF7601efC9E9FE532Bf5284F65d3#code as well please

Done! 😉